tests/pytests/test_kaguya_drivers.py

```diff
@@ -283,5 +283,5 @@ def test_focal2pixel_samples(driver):
 def test_focal2pixel_lines(driver):
     with patch('ale.drivers.selene_drivers.spice.gdpool', return_value=np.array([2])) as gdpool, \
          patch('ale.drivers.selene_drivers.spice.bods2c', return_value=-12345) as bods2c:
-        assert driver.focal2pixel_lines == [0, -1/2, 0]
+        assert driver.focal2pixel_lines == [0, 1/2, 0]
         gdpool.assert_called_with('INS-12345_PIXEL_SIZE', 0, 1)
```

Comment:
You should test with and without the flipping flag.

Reply:
:+1: I put these two checks under one test. It could be converted into two separate tests but I figured it was fine under one test, but I can change it if need be.